lib/routes/account.js

```diff
@@ -252,6 +252,18 @@ module.exports = function (
             tokenVerificationId = undefined
           }
 
+          if (query._createdAt) {
+            // We don't expect this to be set outside the tests so log an error
+            // if we do encounter it, to help debug what's going on.
+            log.error({
+              op: 'account.create.createSessionToken',
+              err: new Error('Unexpected _createdAt query parameter'),
+              _createdAt: query._createdAt,
```

Comment:
Original comment was meant to be consistent with log statements but then I looked in Kibanai and it is not. Feel free to ignore :)

<img width="59" alt="screen shot 2017-02-13 at 11 07 05 am" src="https://cloud.githubusercontent.com/assets/1295288/22895055/2984904e-f1e9-11e6-9547-9aca9eafe3d0.png">